catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/pocketmusic/PocketMusicActivityTest.java

```diff
@@ -128,10 +129,40 @@ protected boolean matchesSafely(TrackView item) {
 		};
 	}
 
+	private static Matcher<View> withRecyclerViewMeasuredSize() {
```

Comment:
please rename to something describing what this does, eg. withSizeGreaterZero, (without looking at this matcher code i would have no idea what it is for when reading withRecyclerViewMeasuredSize in test code)

Reply:
or better **is**SizeGreaterZero, to follow espresso matcher naming conventions.
"with" is used mostly for stuff that takes a parameter that describes something to look for, eg. withText, withId, etc.

while "is" is used for stuff that just matches a condition without parameters, isDisplayed()